### ISSUE TITLE
oxy outputs duplicate access logs when logLevel is set to INFO

### ISSUE THREAD

**Issue by ReillyProcentive** [containous/traefik#2782]
### Do you want to request a *feature* or report a *bug*?
Bug

### What did you do?
Set `logLevel` to `INFO`

### What did you expect to see?
More verbose controller level logs, like when ACME processes run and when backends are added/removed.

### What did you see instead?
Logging entries from vulcand for every single HTTP request that Traefik handles, which essentially duplicates the access logs.

```shell
time="2018-01-29T22:03:00Z" level=info msg="vulcand/oxy/forward/http: Round trip: http://100.104.0.1:8080, code: 200, Length: 0, duration: 2.015332ms tls:version: 303, tls:resume:false, tls:csuite:c02f, tls:server:our.domain" 
time="2018-01-29T22:03:01Z" level=info msg="vulcand/oxy/forward/http: Round trip: http://100.104.0.1:8080, code: 200, Length: 0, duration: 1.991752ms tls:version: 303, tls:resume:false, tls:csuite:c02f, tls:server:our.domain" 
time="2018-01-29T22:03:29Z" level=info msg="vulcand/oxy/forward/http: Round trip: http://100.104.0.1:8080, code: 200, Length: 0, duration: 2.094348ms tls:version: 303, tls:resume:false, tls:csuite:c02f, tls:server:our.domain" 
time="2018-01-29T22:03:29Z" level=info msg="vulcand/oxy/forward/http: Round trip: http://100.104.0.1:8080, code: 200, Length: 0, duration: 2.096226ms tls:version: 303, tls:resume:false, tls:csuite:c02f, tls:server:our.domain" 
time="2018-01-29T22:03:29Z" level=info msg="vulcand/oxy/forward/http: Round trip: http://100.104.0.1:8080, code: 200, Length: 0, duration: 1.984957ms tls:version: 303, tls:resume:false, tls:csuite:c02f, tls:server:our.domain" 
time="2018-01-29T22:12:33Z" level=info msg="vulcand/oxy/forward/http: Round trip: http://100.104.0.1:8080, code: 200, Length: 60, duration: 9.75096ms tls:version: 303, tls:resume:true, tls:csuite:c02f, tls:server:our.domain" 
```

### Output of `traefik version`: (_What version of Traefik are you using?_)
```
Version:      v1.5.1
Codename:     cancoillotte
Go version:   go1.9.3
Built:        2018-01-29_02:14:02PM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
logLevel = "INFO"
checkNewVersion = false
defaultEntryPoints = ["http","https"]
[entryPoints]
  [entryPoints.http]
    address = ":80"
    [entryPoints.http.redirect]
      entryPoint = "https"
  [entryPoints.https]
    address = ":443"
    [entryPoints.https.tls]
      minVersion = "VersionTLS12"
      [[entryPoints.https.tls.certificates]]
        CertFile = "/ssl/wildcard.our.domain/wildcard.our.domain.crt"
        KeyFile = "/ssl/wildcard.our.domain/wildcard.our.domain.key"
  [entryPoints.admin]
    address = ":8080"
[api]
  entrypoint = "admin"
[ping]
  entrypoint = "admin"
[metrics]
  [metrics.prometheus]
    entrypoint = "admin"
    buckets = [0.1,0.3,1.2,5.0]
[lifeCycle]
  requestAcceptGraceTimeout = "30s"
[kubernetes]
```

**Comment by ReillyProcentive** [comment]
Pinging @timoreimann as we discussed this issue in the #support Slack channel. He suggested that the vulcand log entries might be better outputted when `logLevel` is set to `DEBUG`. 

We like to run under `INFO` as the default logging level is not verbose enough for us (we want to be able to see when the controller performs letsencrypt actions or adds/removes a backend) and if we wanted to see this information we would just enable the access log separately.

**Comment by timoreimann** [comment]
@ReillyProcentive thanks for filing the issue.

I'm very much in favor of turning down the noise the vulcand access logs produce. It's basically competing with our very own access logs (which other projects potentially building on top of vulcand will likely have too).

It's primary purpose should be debugging, so the DEBUG log level (or whatever the equivalent is in vulcand) seems much more appropriate.

@emilevauge and the rest of @containous/traefik: any thoughts or concerns?

**Comment by ldez** [comment]
Closed by #2821 and #2847
